java/androidpayload/app/src/com/metasploit/stage/Payload.java

```diff
@@ -13,19 +18,23 @@
 import java.net.Socket;
 import java.net.URL;
 import java.net.URLConnection;
+import java.util.List;
 import java.util.Random;
-import java.util.concurrent.TimeUnit;
 
 import dalvik.system.DexClassLoader;
 
 public class Payload {
 
+    private static final int FLAG_WAKELOCK = 4;
```

Comment:
Is it possible to move these 2 config flags with the others? It would be nice for future people to be able to find all of the config flags in the Config class.